### PR TITLE
Add `deploy_cumulus_distribution` variable to cumulus module

### PR DESCRIPTION
**Pull request by reweeden**
Same as #165 but for the main branch. 

Does the way I did the changelog entries make sense?

### CODE REVIEW

**Comment by lindsleycj** [comment]
@reweeden sorry it's taken me so long to look at this.  In general I have no problem exposing Cumulus variables via CIRRUS, but I think the CIRRUS default should match the Cumulus default.  It looks to me like the Cumulus default is false:

https://github.com/nasa/cumulus/blob/master/tf-modules/cumulus/variables.tf#L608-L612

Is there a reason you think it should be true by default?

**Comment by reweeden** [comment]
That's a good point! It probably does make sense to keep it consistent with the cumulus defaults in CIRRUS-core.

File: cumulus/variables.tf [hunk 1]
```diff
@@ -270,6 +270,12 @@ variable "urs_url" {
   default     = "https://uat.urs.earthdata.nasa.gov"
 }
 
+variable "deploy_cumulus_distribution" {
+  description = "If true, does not deploy the TEA distribution API"
+  type        = bool
+  default     = true
```
Comment (lindsleycj) [inline cumulus/variables.tf:276]: I'll approve this PR as soon as the default value is false as it is in Cumulus

Comment (reweeden) [reply cumulus/variables.tf:276]: Sounds good. I'll have some time today to catch up on all these outstanding PR's

Comment (lindsleycj) [reply cumulus/variables.tf:276]: @reweeden You'll see that I merged the CHANGELOG entry from prod into this branch to resolve a merge conflict due to several unreleased updates.  Once we get them all in place, I'll create a point release of CIRRUS-Core.  I'd like to do that before starting on Cumulus v18.

**Review by lindsleycj — APPROVED** [review]

**Pull request merged**